scoring_harness/challenge_config.template.py

```diff
@@ -0,0 +1,176 @@
+# -----------------------------------------------------------------------------
+#
+# challenge specific code and configuration
+#
+# -----------------------------------------------------------------------------
+# Use rpy2 if you have R scoring functions
+# import rpy2.robjects as robjects
+# import os
+# filePath = os.path.join(
+#     os.path.dirname(os.path.abspath(__file__)), 'getROC.R')
+# robjects.r("source('%s')" % filePath)
+# AUC_pAUC = robjects.r('GetScores')
+
+# Configuring them here as a list will save a round-trip to the server
+# every time the script starts and you can link the challenge queues to
+# the correct scoring/validation functions.
+# Predictions will be validated and scored
+
+
+def no_score(submission_path, goldstandard_path):
+    '''
+    Use this function for evaluation queues that don't need scoring
+    Args:
+        submission_path:  Path to submission file
+        goldstandard_path: Path to truth file
+    '''
+    pass
+
+
+def validate_func(submission_path, goldstandard_path):
+    '''
+    Validate submission.
+
+    MUST USE ASSERTION ERRORS!!!
```

Comment:
this should be changed. Assertion errors should only be used for internal purposes, not for error tracking and handling. If you need a specific error type, create a new error class.